packages/common/src/helper/jsonToArray.ts

```diff
@@ -0,0 +1,11 @@
+// used for converting json values to an array to allow map fn to work
+export const jsonToArray = (json?: any) => {
```

Comment:
isn't it same with Object.values(json)?

Reply:
yes, I changed it to Object.values since I have no need for the keys, good catch!